lib/internal/perf/observe.js

```diff
@@ -233,11 +242,7 @@ ObjectDefineProperties(PerformanceObserverEntryList.prototype, {
 });
 
 function createPerformanceObserverEntryList(entries) {
```

Comment:
This function is only called in one place, perhaps this can just be inlined at that call site instead?

Reply:
Make sense